### DISCUSSION TITLE
psql compatible embedded database

### DISCUSSION THREAD

**Discussion by v217** [EvgSkv/logica#22] [Ideas]
I've read that https://duckdb.org/ uses the same parser as psql.
I don't know about embedded MonetDB/e.
So for newcomers an embedded database is far easier to get started with than postgresql.
This might be a temporary solution till sqlite is fully supported.
Thanks!

**Comment by EvgSkv** [comment]
Thanks for pointing this out!
It appears there are some differences in DuckDB and PostgreSQL syntax, as query 
```
SELECT x FROM UNNEST([1,2,3]) AS x
```
gave me a syntax error related to array notation in DuckDB. I would like to look at it more carefully though.

**Comment by v217** [comment]
Thank you, I confounded 'same parser' with 'same functionality'.
Monetdb supports the `UNNEST` command, but I don't know enough SQL to run the examples. There are lots of differences eg. `<>` instead of `!=`, etc ...

**Comment by EvgSkv** [comment]
Core of Logica functionality now works on SQLite, which requires no setup.

**Reply by v217** [reply]
That's absolutely great news! I think a lot of people will be happy about this!
